docs(RIF Relay): change the RIF Relay documentation to show the usage of the new version of the libraries (#873)
rsksmart/devportal: guides/rif-relay/integrate.md

Users can interact with the RIF Relay Server directly or indirectly. For the latter, a user can communicate with a RIF Relay Server through a RIF Relay Client. A RIF Relay Client knows the addresses of different RIF Relay Servers and it can send on-chain requests to any one of them. The RIF Relay Client then sends the transaction to be sponsored to the RIF Relay Server via HTTP request.

In any case, you'll need to have the server installed and running. To achieve this please refer to the following guides:
1. [RIF Relay Installation Requirements](/guides/rif-relay/installation-requirements/)
2. [RIF Relay Deployment](/guides/rif-relay/deployment/)

### RIF Relay Client

> The `Relay Client` is a component from the RIF Relay Client library that helps to build a relay request, search for a server and send the request via http protocol. 

Reviewer: ```suggestion
The `RelayClient` class from the RIF Relay Client library assists in building a relay request, searching for an available server and sending the request via http protocol. 
```

> To create a `Relay Client` we need to follow the next steps:

Reviewer: ```suggestion
To create a `Relay Client` we need to follow these steps:
```

> 1. Set the configuration.
> 2. Set the ethers provider.
> 3. Create intances. 

Reviewer: ```suggestion
1. Set configuration
2. Set (ethers) provider
3. Create intance
```

```typescript
import {
  RelayClient,
  setEnvelopingConfig,
  setProvider,
} from '@rsksmart/rif-relay-client';

>   setEnvelopingConfig({
>     chainId: <CHAIN_ID>,
>     preferredRelays: <SERVER_URL_ARRAY>,
>     relayHubAddress: <RELAY_HUB_ADDRESS>,
>     deployVerifierAddress: <DEPLOY_VERIFIER_ADDRESS>,
>     relayVerifierAddress: <RELAY_VERIFIER_ADDRESS>,
>     smartWalletFactoryAddress: <SMART_WALLET_FACTORY_ADDRESS>
>   });

Reviewer: What if we add descriptions here so the user doesn't have to scroll up and down?

```suggestion
  setEnvelopingConfig({
    chainId: <CHAIN_ID>, // Identifies a network to interact with.
    preferredRelays: <SERVER_URL_ARRAY>, // An array of relay server URL strings that the RelayClient can interact with.
    relayHubAddress: <RELAY_HUB_ADDRESS>, // The relay hub contract address.
    deployVerifierAddress: <DEPLOY_VERIFIER_ADDRESS>, // The deploy verifier contract address.
    relayVerifierAddress: <RELAY_VERIFIER_ADDRESS>, // The relay verifier contract address.
    smartWalletFactoryAddress: <SMART_WALLET_FACTORY_ADDRESS> // The smart wallet factory contract address.
  });
```

Author: @jurajpiar Since we specify the variable meaning below, I would avoid duplication by describing them in the code snippet also.

Reviewer: I agree, which is why I suggested to only describe it here instead of bellow so that the user doesn't have to switch between different sections to use this. But ofc, it's up to you :)

  setProvider(ethersProvider);
  
  const relayClient =  new RelayClient();
```

Where variables are:

>   * **CHAIN_ID**: The chain id that identify the network that the client will be interacting.

Reviewer: ```suggestion
  * **CHAIN_ID**: Identifies a network to interact with.
```

>   * **SERVER_URL_ARRAY**: An array of relay server URLs that the client can interact with.

Reviewer: ```suggestion
  * **SERVER_URL_ARRAY**: An array of relay server URL strings that the RelayClient can interact with.
```
